### PR TITLE
Issue 728 IDLE/Pressure tags shall not be applied to the paused runs 

### PR DESCRIPTION
**Pull request by Wedds**
This PR solves issue #728

IDLE and PRESSURE tags are dropped when a job is being paused.

- Drop utilization tags when a run state is changing to PAUSED.
- Unit tests to prove tags are being removed.

### CODE REVIEW

File: api/src/test/java/com/epam/pipeline/manager/docker/DockerContainerOperationManagerTest.java [hunk 1]
```diff
@@ -29,9 +38,11 @@
     private static final String INSUFFICIENT_INSTANCE_CAPACITY = "InsufficientInstanceCapacity";
     private static final String NODE_NAME = "node-1";
     private static final String NODE_ID = "node-id";
+    private static final String STUB_STRING = "";
```
Comment (mzueva) [inline api/src/test/java/com/epam/pipeline/manager/docker/DockerContainerOperationManagerTest.java]: Could you please rename `STUB_STRING` to `TEST_TAG` with some value, to make test a little bit more clear.

File: api/src/main/java/com/epam/pipeline/manager/docker/DockerContainerOperationManager.java [hunk 2]
```diff
@@ -367,5 +369,12 @@ private void addRunLog(final PipelineRun run, final String logMessage, final Str
                 .build();
         runLogManager.saveLog(runLog);
     }
+
+    private void removeUtilizationLevelTags(final PipelineRun run) {
+        Stream.of(ResourceMonitoringManager.UTILIZATION_LEVEL_LOW,
```
Comment (mzueva) [inline api/src/main/java/com/epam/pipeline/manager/docker/DockerContainerOperationManager.java:373]: We also need to persist these changes to db using dao

**Review by mzueva — CHANGES_REQUESTED** [review]

File: api/src/test/java/com/epam/pipeline/manager/docker/DockerContainerOperationManagerTest.java (hunk 1 above)
Comment (Wedds) [reply api/src/test/java/com/epam/pipeline/manager/docker/DockerContainerOperationManagerTest.java]: Done

File: api/src/main/java/com/epam/pipeline/manager/docker/DockerContainerOperationManager.java (hunk 2 above)
Comment (Wedds) [reply api/src/main/java/com/epam/pipeline/manager/docker/DockerContainerOperationManager.java:373]: Done

**Review by mzueva — APPROVED** [review]

File: api/src/main/java/com/epam/pipeline/manager/docker/DockerContainerOperationManager.java [hunk 3]
```diff
@@ -257,9 +259,9 @@ public void pauseRun(PipelineRun run) {
             kubernetesManager.deletePod(run.getPodId());
             cloudFacade.stopInstance(instance.getCloudRegionId(), instance.getNodeId());
             kubernetesManager.deleteNode(instance.getNodeName());
-
+            removeUtilizationLevelTags(run);
             run.setStatus(TaskStatus.PAUSED);
-            runManager.updatePipelineStatus(run);
+            runManager.updateRunInfo(run);
```
Comment (mzueva) [inline api/src/main/java/com/epam/pipeline/manager/docker/DockerContainerOperationManager.java]: Could you please move `removeUtilizationLevelTags(run)` and `runManager.updateRunInfo(run)` calls to line 241 (just after `runManager.updateServiceUrl`).  Replace calls to `updatePodIP` and `updateServiceUrl` with new `updateRunInfo` method.

**Review by mzueva — CHANGES_REQUESTED** [review]

File: api/src/main/java/com/epam/pipeline/manager/docker/DockerContainerOperationManager.java (hunk 3 above)
Comment (Wedds) [reply api/src/main/java/com/epam/pipeline/manager/docker/DockerContainerOperationManager.java]: Done

**Review by mzueva — APPROVED** [review]

**Pull request merged**